### ISSUE TITLE
parsing wrong path when test by list of path

### ISSUE THREAD

**Issue by chiuchiu328** [promptfoo/promptfoo#517]
I was running npx promptfoo@latest eval --no-cache 
I got error when I test by using list of path.  When I did
```
tests: 
  - test/test1.csv
```
or
```
tests: ['test/test1.csv']
```  
I got 
<img width="1261" alt="Screenshot 2024-03-01 at 9 25 30 PM" src="https://github.com/promptfoo/promptfoo/assets/9929375/fa6668f3-6bac-44d3-9b54-3a4bdaf2389f">

if I use
```
tests: test/test1.csv
```
then it worked perfectly.

here is my all config
```
description: 'My first eval'

prompts:
  prompt/one_prompt_test.txt: onePromptTest
  prompt/some_prompt_test.txt: somePromptTest
  prompt/four_prompt_test.txt: fourPromptTest

providers:
  - id: 'python:provider/my_script.py'
    config:
      additionalOption: 123
    prompts: fourPromptTest
  - id: 'python:provider/new_script.py'
    config:
      additionalOption: 123
    prompts: [onePromptTest, somePromptTest]

tests: ['test/test1.csv']
```

I am running it on macOS 
and node.js v21.6.2
python:3.11

**Comment by typpo** [comment]
#520 should resolve this issue.  It will be in the next release

The problem was that `csv` files were not supported in the `tests` array.

**Comment by typpo** [comment]
Fix released in 0.44.0
